mk/build.sh

```diff
@@ -72,14 +73,57 @@ XENADMIN_DIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 source ${XENADMIN_DIR}/mk/declarations.sh
 
+if test -z "${XC_BRANDING}"; then XC_BRANDING=citrix; fi
```

Comment:
This PR restores here code that was recently removed or moved to different files. Please compare with the current state of this file on master and remove what's not there.